### ISSUE TITLE
TestDateTime.GetAsTimeStamp test fails

### ISSUE THREAD

**Issue by candrews** [xbmc/xbmc#17909]
### Describe the bug
Running the tests while building Kodi 19 / current master https://github.com/xbmc/xbmc/commit/875a8f65805b82fa95e902ec194f8a4673a9b3e1 results in 1 test failure:
```
        468 - TestDateTime.GetAsTimeStamp (Failed)
```
## Expected Behavior
Here is a clear and concise description of what was expected to happen:

No tests should fail.

## Actual Behavior

There is a test failure.

### To Reproduce

Run the tests as part of the Kodi build.

### Debuglog

n/a (this is a build / test issue, not a runtime one)

### Screenshots 
Here are some links or screenshots to help explain the problem:
n/a (this is a build / test issue, not a runtime one)

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [X] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Gentoo/Linux, amd64
 - Kodi version: master

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by candrews** [comment]
@lrusak Since you wrote this test, perhaps you could share some thoughts on what may be wrong and/or how to fix it?

**Comment by lrusak** [comment]
@candrews the reason I wrote these tests in the first place is to set a baseline for when I finish implementing std::chrono and std::date changes. It's mostly done but I haven't looked at it in a couple months do to some time constraints.

tl;dr disable the test if it's not working.
